components/org.wso2.carbon.identity.sts.passive/src/main/java/org/wso2/carbon/identity/sts/passive/utils/RequestProcessorUtil.java

```diff
@@ -260,21 +265,17 @@ public static String[] getKeyStoreAliasAndKeyStorePassword(
 
         String[] aliasAndPassword = new String[2];
 
-        String keyStorePassword;
-        String keyAlias;
-
-        boolean isSuperTenantDomain = (MultitenantConstants.SUPER_TENANT_ID == tenantId);
-        if (isSuperTenantDomain) {
-            keyAlias = serverConfig.getFirstProperty(KEY_ALIAS_KEY);
-            keyStorePassword = serverConfig.getFirstProperty(KEY_STORE_PASSWORD_KEY);
-        } else {
-            String keyStoreName = generateKSNameFromDomainName(tenantDomain);
-            keyAlias = tenantDomain;
-            keyStorePassword = KeyStoreManager.getInstance(tenantId).getKeyStorePassword(keyStoreName);
-        }
-
-        aliasAndPassword[0] = keyAlias;
-        aliasAndPassword[1] = keyStorePassword;
+        // Key Alias
```

Comment:
if this comment is required, please make it meaningful.

Reply:
Removed in https://github.com/wso2-extensions/identity-inbound-auth-sts/pull/160/commits/1ac1853d634401e7950863b9f292c36b9aa5a203